ingest/src/main/java/com/connexta/ingest/adaptors/S3Adaptor.java

```diff
@@ -52,9 +51,10 @@ public S3Adaptor(S3StorageConfiguration configuration) {
     s3Endpoint = configuration.getS3Endpoint();
     s3Region = configuration.getS3Region();
     s3BucketQuarantine = configuration.getS3BucketQuarantine();
+
+    initializeS3();
```

Comment:
It's weird that we have to do this because that's what `@PostConstruct` is supposed to be doing. 😞

Reply:
I agree, I think we can actually move back to `@PostConstruct`